### PR TITLE
feat/POR-143: Recommended Profiles

### PR DESCRIPTION
**Pull request by juanarancibia**
[Linear Card](https://linear.app/portalo/issue/POR-143/recommended-profiles-crecimiento)

### CODE REVIEW

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 import HomeCardsList from './HomeCardList';
 
 const Home: FC = () => {
   return (
-    <div className="space-y-4">
+    <div className="space-y-6">
       <Onboarding />
 
-      <ResponsiveDialog
+      {/* <ResponsiveDialog
```
Comment (pcorrado-np) [inline src/components/home/Home.tsx:10]: Remove this

File: src/components/profiles/ProfileWidget.tsx [hunk 2]
```diff
@@ -20,27 +22,36 @@ const ProfileWidget = () => {
...
+    <PlainCard
+      title={<TypographyH5> {profilesTitle}</TypographyH5>}
+      titleIcon={<UserRound />}
+      hideSeparator
+      ctaTitle="Add Profile"
+      ctaIcon={<Plus />}
+      onCtaClick={() => router.push(ROUTES.APP_CREATE_PROFILE)}
+      content={
+        hasProfiles ? (
+          <>
```
Comment (pcorrado-np) [inline src/components/profiles/ProfileWidget.tsx:34]: The Fragment isn't needed

**Review by pcorrado-np — APPROVED** [review]

File: src/components/folders/FolderShare.tsx [hunk 3]
```diff
@@ -14,6 +15,15 @@ import { useRouter } from 'next/navigation';
 import { FC, useMemo } from 'react';
 import { useRecoilValue } from 'recoil';
 
+const animationConfig = {
```
Comment (Braifz) [inline src/components/folders/FolderShare.tsx:18]: I think this animation could be made with TailwindCSS. 

https://tailwindcss.com/docs/animation#using-custom-values

File: src/core/components/PlainCard.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  content: ReactElement;
+  title: ReactElement;
+  titleIcon: ReactElement;
+  ctaTitle?: string;
+  ctaIcon?: ReactElement;
+  onCtaClick?: () => void;
+  hideSeparator?: boolean;
+}
+
+const PlainCard: FC<PlainCardProps> = ({
```
Comment (Braifz) [inline src/core/components/PlainCard.tsx:15]: I think the naming could be better. Maybe something like "BaseCard" or "FeatureCard"?

**Review by Braifz — APPROVED** [review]
Great! I leave a few comments. 👏🏼

File: src/components/folders/FolderShare.tsx (hunk 3 above)
Comment (juanarancibia) [reply src/components/folders/FolderShare.tsx:18]: https://linear.app/portalo/issue/POR-172/change-framer-motion-to-tailwind-animations

File: src/core/components/PlainCard.tsx (hunk 4 above)
Comment (juanarancibia) [reply src/core/components/PlainCard.tsx:15]: It's not even a card. Will change the naming and structure of the component in the future

**Pull request merged**